cfgov/v1/tests/test_password_policy.py

```diff
@@ -81,12 +81,12 @@ def test_cant_set_password_if_locked(self):
 
 class TestPasswordReusePolicy(TestWithUser):
     def test_can_set_new_password(self):
-        user = self.get_user('password1')
+        user = self.get_user(last_password='password1')
         password_policy.validate_password_history(user, 'password2')
 
     def test_cant_reuse_password(self):
         password = 'password'
-        user = self.get_user(password)
+        user = self.get_user(last_password=password)
         self.assertRaises(
```

Comment:
Actually, nevermind - I think it's slightly clearer to use the string but I see why you used a variable in `test_cant_resuse_password` and not the other, since it's used twice.